pkg/ansible/controller/reconcile.go

```diff
@@ -62,6 +62,7 @@ type AnsibleOperatorReconciler struct {
 }
 
 // Reconcile - handle the event.
+//nolint:gocyclo, gosec
```

Comment:
This is not enabled in the CI.
To test the same lint checks locally which are made in the CI you can run `./hack/tests/check-lint.sh ci`

Reply:
i have pushed the changes for the checks on the ci only